Vagrantfile

```diff
@@ -79,6 +79,24 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  ## Provider-specific settings, libvirt
+  config.vm.provider :libvirt do |v,override|
+    override.vm.box = "fedora21"
+    override.vm.box_url = "http://citozin.com/fedora21.box"
```

Comment:
The boxes used for the other providers are official distribution images. If we're going to add some Fedora/RH/Centos boxes, could we use boxes from the projects themselves?